ocs_ci/helpers/helpers.py

```diff
@@ -1174,13 +1188,14 @@ def get_pattern_time(log, snapname, pattern):
             str: string of the pattern timestamp in the log, if not found None
 
         """
+        this_year = str(datetime.datetime.now().year) + " "
         for line in log:
             if re.search(snapname, line) and re.search(pattern, line):
-                return line.split(" ")[1]
+                return this_year + " ".join(line.split(" ")[0:2])
```

Comment:
I think we should use f-strings instead of string concat. Maybe we can use a variable for `" ".join(line.split(" ")[0:2]` as well so it's more clear via the code what this represents. As it stands I don't really know what this is supposed to be. I would also address other similar lines in the same way.